src/jsx/ChatBox/test_message.jsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx React.DOM */
```

Comment:
Is this how we are going to label tests? With `test_<name of file>`? Just curious because then we can set this as the standard.